pkg/cfn/cfn.go

```diff
@@ -51,7 +52,8 @@ func (c Cfn) CreateStackAndGetResources(availabilityZones []*ec2.AvailabilityZon
 	stackName *string, template string) (vpcId *string, subnetIds []string, instanceId *string,
 	stackResources []*cloudformation.StackResource, err error) {
 	if stackName == nil {
-		stackName = aws.String(DefaultStackName)
+		stackIdentifier := uuid.New()
+		stackName = aws.String(fmt.Sprintf("%s%s", DefaultStackName, stackIdentifier))
```

Comment:
This change is causing a failure in the unit tests: they never terminate because they're [looking](https://github.com/awslabs/aws-simple-ec2-cli/blob/main/pkg/cfn/cfn.go#L142) for a stack [called](https://github.com/awslabs/aws-simple-ec2-cli/blob/main/pkg/cfn/cfn.go#L29) `simple-ec2` when the actual name we requested is more like `simple-ec2e77446f0-ca6b-469a-8eb0-6d7163604fda`. Working on a fix.

Reply:
Resolved in e5d4ccb771b722373925388017802b195fac6a21.